src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -232,17 +236,18 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
     }))
   }
 
-  handleSubmit = () => {
+  handleSubmit = async () => {
     const { selectedFeedbackOption, comment } = this.state
-    this.postFeedbackData(this.getFeedbackData(selectedFeedbackOption, comment))
-    this.props.onSubmit()
+    await this.postFeedbackData(this.getFeedbackData(selectedFeedbackOption, comment))
   }
 
   render () {
-    const { closeFeedbackModal, isPositiveRatingSelected, theme } = this.props
+    const { closeFeedbackModal, isPositiveRatingSelected, theme, feedbackSent } = this.props
+
     return <FeedbackBox onFeedbackOptionChanged={this.handleFeedbackOptionChanged}
                         onCommentChanged={this.handleCommentChanged}
                         onSubmit={this.handleSubmit}
```

Comment:
Typing feels off here. `onSubmit` is typed as `(value?: string) => void` but `this.handleSubmit` does not expect any parameter.

Reply:
The prop `onSubmit` with its optional output value of `feedbackSent` has moved to `postFeedbackData`. It only generates an output depending on the request response. Imo there is no need to pass it here.

Reply:
@Taggotty is right. See my comment in `FeedbackBox`, it is not necessary to call `onSubmit` with `feedbackSent` in the `FeedbackBox`.